src/components/accession2/Accession2PlantSiteDetails.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import strings from 'src/strings';
+import { Link, Grid, Box, useTheme } from '@mui/material';
+import { AccessionPostRequestBody } from 'src/api/accessions2/accession';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import Textfield from 'src/components/common/Textfield/Textfield';
+import { Select } from '@terraware/web-components';
+import { ACCESSION_2_COLLECTION_SOURCES } from 'src/types/Accession';
+
+type Accession2PlantSiteDetailsProps = {
+  record: AccessionPostRequestBody;
+  onChange: (id: string, value?: any) => void;
+};
+
+export default function Accession2PlantSiteDetails(props: Accession2PlantSiteDetailsProps): JSX.Element {
+  const { record, onChange } = props;
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [numPlants, setNumPlants] = useState<string>('');
+  const { isMobile } = useDeviceInfo();
+  const theme = useTheme();
+
+  const gridSize = () => (isMobile ? 12 : 6);
+
+  const onChangeNumPlants = (value: any) => {
+    const numPlantsStr = value as string;
+    setNumPlants(numPlantsStr);
+    // tslint:disable-next-line: no-unnecessary-initializer
+    let min = undefined;
```

Comment:
Why are we initializing this as undefined? And not just write `let min; let max;` ?

Reply:
This is to express that we are setting undefined intentionally in the onChange.

Reply:
removed it